src/jit/compiler.h

```diff
@@ -2276,6 +2276,10 @@ public :
     unsigned short      lvaTrackedCount;    // actual # of locals being tracked
     unsigned            lvaTrackedCountInSizeTUnits;  // min # of size_t's sufficient to hold a bit for all the locals being tracked
 
+#ifdef FEATURE_MULTIREG_STRUCT_ARGS
```

Comment:
Shouldn't the #ifdef be FEATURE_UNIX_AMD64_STRUCT_PASSING?  We seems to be mixing different things here...